cartridges/app_storefront_custom/cartridge/templates/default/content/assetsPage.isml

```diff
@@ -0,0 +1,11 @@
+<isif condition="${pdict.currentCustomer.getProfile().firstName}">
```

Comment:
This check should be done in the controller, also additional null check is needed.